packages/electron/package.json

```diff
@@ -14,12 +14,22 @@
     "url": "https://github.com/eclipse-theia/theia/issues"
   },
   "homepage": "https://github.com/eclipse-theia/theia",
+  "files": [
+    "native",
+    "lib",
+    "src",
+    "*.js"
+  ],
   "bin": {
     "electron": "electron-cli.js",
     "electron-codecs-test": "electron-codecs-test.js",
     "electron-replace-ffmpeg": "electron-replace-ffmpeg.js"
   },
+  "devDependencies": {
+    "@theia/cli": "^1.2.0"
```

Comment:
why do we need it here?

Reply:
I will check whether we need it or it's a leftover.